css/styles.css

```diff
@@ -121,3 +120,98 @@ button {
     width: 100%;
   }
 }
+
+.hide {
+  display: none;
+}
```

Comment:
There is already a `hidden` class rule in `layout.css`, you should remove this, and just use that `hidden` class